exploring_UV_extinction_curves_pt2/Exploring_UV_extinction_curves_pt2.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>I think it would be great to include this link back in the first introduction, and let the reader know that we will be reproducing the work in the paper! (Also shouldn't it be Gordon 2002?) It is really exciting for someone new to a field to have that "I did it!" moment like when they create the plot above. To have that moment they need a picture in their mind of what they were trying to create. Were these sources also chosen from the paper? That would be good to cite.</p><p> </p><p>Also, it may be more useful to send the link to the ADS page rather than the arxiv PDF.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>